src/components/Template/Template.stories.mdx

```diff
@@ -0,0 +1,410 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+
+import {
+  Template,
+  TemplateHeader,
+  TemplateContentBreakout,
+  TemplateContent,
+  TemplateContentTop,
+  TemplateContentPrimary,
+  TemplateContentSidebar,
+  TemplateFooter,
+  TemplateAppContainer,
+} from "./Template";
+import Placeholder from "../Placeholder/Placeholder";
+import { getCategory } from "../../utils/componentCategories";
+
+<Meta
+  title={getCategory("Template")}
+  component={TemplateAppContainer}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=11982%3A47778",
+    },
+  }}
+/>
+
+# Template
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.3.6`    |
+| Latest            | `0.25.1`   |
+
+The DS "template" components are a set of components that work together to
+render a consistent mobile and desktop layout. More information on how they
+should be composed below.
+
+Basic component structure:
```

Comment:
We haven't decided on this yet and now is the time to do so. Should we suggest the `TemplateAppContainer` or the individual components?

I'm okay with recommending `TemplateAppContainer` but it would update a lot of this PR, which I'm okay with doing.